renderdoc/driver/gl/gl_pixelhistory.cpp

```diff
@@ -276,13 +400,228 @@ void QueryFailedTests(WrappedOpenGL *driver, GLPixelHistoryResources &resources,
       }
     }
 
-    history[i].scissorClipped = failedTest == OpenGLTest::ScissorTest;
-    history[i].stencilTestFailed = failedTest == OpenGLTest::StencilTest;
-    history[i].depthTestFailed = failedTest == OpenGLTest::DepthTest;
-    history[i].backfaceCulled = failedTest == OpenGLTest::FaceCulling;
+    for(; historyIndex < history.size() && modEvents[i].eventId == history[historyIndex].eventId;
```

Comment:
Can we keep this how it was before?
I'm pretty sure we don't duplicate the history elements at this point any more.